src/libraries/Native/native-binplace.proj

```diff
@@ -26,4 +18,7 @@
       <FileWrites Include="@(BinPlaceItem)" />
     </ItemGroup>
   </Target>
+
+  <!-- Workaround for https://github.com/microsoft/MSBuildSdks/issues/120. -->
+  <Target Name="CreateManifestResourceNames" />
```

Comment:
cc @jeffkl

Reply:
I can probably get this fixed today if you want to wait for it

Reply:
Sure, thanks :)

Reply:
This will probably need to wait another day anyway, sure :)